lib/broccoli/ember-app.js

```diff
@@ -152,21 +152,21 @@ class EmberApp {
     this._defaultPackager = new DefaultPackager({
       env: this.env,
       name: this.name,
+      autoRun: this.options.autoRun,
```

Comment:
is there a reason to change the order of arguments?

Reply:
I changed the order so it's same to the order in the default-packager https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/default-packager.js#L125